DQM/CastorMonitor/interface/CastorDigiMonitor.h

```diff
@@ -44,6 +44,12 @@ class CastorDigiMonitor: public CastorBaseMonitor {
   MonitorElement *hSector;
   MonitorElement* hdigisize;
 //  MonitorElement* hBunchOcc;
+
+  int TS_MAX = 10;
```

Comment:
I am not the author of this, so I want to limit my changes to just removing static variables.